.github/workflows/run-tests.yml

```diff
@@ -7,6 +7,7 @@ jobs:
       VAULT_ADDR: http://0.0.0.0:8200
       VAULT_DEV_ROOT_TOKEN_ID: testtoken
       EXPECTED_SECRET_VALUE: Hashi123
+      SOME_ENV_VAR: my-password
```

Comment:
What do we use this for? Should we rename this to something more descriptive?

Reply:
Realizing now how silly it looks in the Github yaml haha.... this is used in the userpass login where it says:

```
	userpassAuth, err := auth.NewUserpassAuth("my-user", &auth.Password{FromEnv: "SOME_ENV_VAR"})
```

How bout I rename it `PASSWORD_ENV_VAR`? I just wanted to make it obvious at a glance that they're supposed to type the name of an environment variable there, not put the actual password value (since they're using the `FromEnv` field).

Reply:
Right! I think that makes sense. I was thinking just for maintainability when we go to look at this again, we know what it's for haha